rcl/include/rcl/node.h

```diff
@@ -581,59 +582,9 @@ rcl_get_disable_loaned_message(bool * disable_loaned_message);
  */
 RCL_PUBLIC
 RCL_WARN_UNUSED
-rcl_ret_t rcl_node_type_description_service_init(rcl_node_t * node);
-
-/// Finalizes the node's ~/get_type_description service.
-/**
- * This function finalizes the node's private ~/get_type_description service.
- *
- * <hr>
- * Attribute          | Adherence
- * ------------------ | -------------
- * Allocates Memory   | No
- * Thread-Safe        | No
- * Uses Atomics       | No
- * Lock-Free          | Yes
- *
- * \param[in] node the handle to the node whose type cache should be initialized
- * \return #RCL_RET_OK if service was deinitialized successfully, or
- * \return #RCL_RET_INVALID_ARGUMENT if any arguments are invalid, or
- * \return #RCL_RET_SERVICE_INVALID if the service is invalid, or
- * \return #RCL_RET_NODE_INVALID if the node is invalid, or
- * \return #RCL_RET_ERROR if an unspecified error occurs.
- */
-RCL_PUBLIC
-RCL_WARN_UNUSED
-rcl_ret_t rcl_node_type_description_service_fini(rcl_node_t * node);
-
-
-/// Returns a pointer to the node's ~/get_type_description service.
-/**
- * On success, sets service_out to the initialized service.
- * rcl_node_type_description_service_init must be called before this.
- *
- * <hr>
- * Attribute          | Adherence
- * ------------------ | -------------
- * Allocates Memory   | No
- * Thread-Safe        | No
- * Uses Atomics       | No
- * Lock-Free          | Yes
- *
- * \param[in] node the handle to the node
- * \param[out] service_out Handle to pointer that will be set
- * \return #RCL_RET_OK if valid service was returned successfully, or
- * \return #RCL_RET_NODE_INVALID if node is invalid, or
- * \return #RCL_RET_INVALID_ARGUMENT if any arguments are invalid, or
- * \return #RCL_RET_NOT_INIT if the service hasn't yet been initialized, or
- * \return #RCL_RET_ERROR if an unspecified error occurs.
- */
-RCL_PUBLIC
-RCL_WARN_UNUSED
-rcl_ret_t rcl_node_get_type_description_service(
-  const rcl_node_t * node,
-  rcl_service_t ** service_out);
-
+rcl_ret_t rcl_node_type_description_service_init(
+  rcl_service_t * service,
```

Comment:
doc section needs to be update accordingly?

```c
 * \param[in] service the handle to the type description service to be initialized
 * \param[in] node the handle to the node whose type cache should be initialized
```

Reply:
[6f7da3a](https://github.com/ros2/rcl/pull/1112/commits/6f7da3a16387b59e799e33463cb6daecb4de7a19)